mep/people/management/commands/export_locations.py

```diff
@@ -0,0 +1,119 @@
+"""
+Manage command to export member data for use by others.
+
+Generates a CSV and JSON file including details on every library member in the
+database, with summary details and coordinates for associated addresses.
+
+"""
+
+from collections import OrderedDict
+from django.db.models import Prefetch
+from mep.common.management.export import BaseExport
+from mep.common.templatetags.mep_tags import domain
+from mep.common.utils import absolutize_url
+from mep.people.models import Location, Person
+from mep.accounts.models import Account, Address
+from mep.accounts.partial_date import DatePrecisionField
+
+
+class Command(BaseExport):
+    """Export member data."""
+
+    help = __doc__
+
+    model = Address
+
+    csv_fields = [
+        "id",  # including "id" to store slug for exports,
```

Comment:
locations/addresses don't have IDs, as far as I can remember (other than db id, which we should not include in the exports)

do you think we need public ids for these? I was assuming we would not

Reply:
Because we don't have URIs for locations or creators, I am including the slug, which we do have, for all 4 exports, but calling it "id" instead of slug. This allows csv-users to easily merge the datasets. That or something like it would be my recommendation but up to you.

Reply:
But we could leave out an "id" field for locations since Location/Address doesn't have a slug, and since the output csv should still be mergeable with the others on `member`

Reply:
- I agree that locations / address data export does not need IDs, just needs to have a member uri to join with the members dataset
- I can't remember if I suggested: I think we should include member_uri in the creators dataset, so that any authors who are also library members can be easily identified and the author and member records  can be connected (this seems better than a boolean indicating membership)
- We need to keep URIs where we have them (which is everywhere but creators, right?), and merging/joining can always be done based on the URIs. I always figured short ids could be generated from URIs downstream where they are useful; not sure of the balance between redundancy and convenience in the published datasets
- How are you proposing to support merging book and creator data? We're using authorized names where we have them, so it should be possible to merge based on those, which I think we do include in both datasets. What do you think about a csv explicitly documenting the relationships? I don't love it personally but it's also more explicit, and would be more useful for some kinds of questions.